requirements-dev.txt

```diff
@@ -12,7 +12,7 @@ pytest-cov>=4.0.0
 pytest-django>=4.8.0
 pytest-fail-slow>=0.3.0
 pytest-rerunfailures>=11
-pytest-sentry>=0.1.11
+https://github.com/getsentry/pytest-sentry/archive/refs/heads/szokeasaurusrex/sdk-2.0-support.zip
```

Comment:
I think you can not add this here. You need to add it here: https://github.com/getsentry/sentry/blob/master/requirements-getsentry-overrides.txt#L0-L1 (and let the this line be the existing `pytest-sentry>=0.1.11`.